docs_src/gitbuch_10.adoc

```diff
@@ -170,206 +175,216 @@ $ *git checkout -- [TAB]*
-vervollständigt, sobald sie eindeutig ist. Existiert nur der Branch
-`feature`, führt die Eingabe von `git checkout fe[TAB]`
-dazu, dass `fe` vervollständigt wird; der Befehl `git
-  checkout feature` steht dann auf der Kommandozeile -- drücken Sie
-auf Enter, um den Befehl auszuführen. Nur wenn die Eingabe nicht
-eindeutig ist, zeigt die Bash die möglichen Vervollständigungen an.
+////
+Wie bei der Bash-Completion üblich, wird die Eingabe automatisch vervollständigt, sobald sie eindeutig ist. Existiert nur der Branch `feature`, führt die Eingabe von `git checkout fe[TAB]` dazu, dass `fe` vervollständigt wird; der Befehl `git checkout feature` steht dann auf der Kommandozeile -- drücken Sie auf Enter, um den Befehl auszuführen. Nur wenn die Eingabe nicht eindeutig ist, zeigt die Bash die möglichen Vervollständigungen an.
+////
+
+As usual with bash completion, the input is automatically completed when it is unique. If only the Branch `feature` exists, typing `git checkout fe[TAB]` will cause `fe` to be completed; the command `git checkout feature` will then appear on the command line - press Enter to execute the command. Only when the input is ambiguous does the bash display the possible completions.
 
 [[sec.bash-prompt]]
 === Prompt
+// === Prompt
 
-Neben der Completion gibt es ein weiteres Script, um Infos über das
-Git-Repository im Prompt anzuzeigen. Dafür müssen Sie die Datei
-`contrib/completion/git-prompt.sh` laden (ggf. ist diese auch von Ihrer
-Distribution installiert, z.B. unter `/usr/lib/git-core/git-sh-prompt`).
-Setzen Sie anschließend -- wie in folgendem Beispiel -- einen
-Aufruf der Funktion `__git_ps1` in die Variable `PS1`
-ein.  Als Argument nimmt die Funktion einen sogenannten
-_Format-String-Ausdruck_ entgegen -- d.h. die Zeichenfolge
-`%s` wird durch Git-Infos ersetzt, alle anderen Zeichen werden
-übernommen.
+////
+Neben der Completion gibt es ein weiteres Script, um Infos über das Git-Repository im Prompt anzuzeigen. Dafür müssen Sie die Datei `contrib/completion/git-prompt.sh` laden (ggf. ist diese auch von Ihrer Distribution installiert, z.B. unter `/usr/lib/git-core/git-sh-prompt`). Setzen Sie anschließend -- wie in folgendem Beispiel -- einen Aufruf der Funktion `__git_ps1` in die Variable `PS1` ein. Als Argument nimmt die Funktion einen sogenannten _Format-String-Ausdruck_ entgegen -- d.h. die Zeichenfolge `%s` wird durch Git-Infos ersetzt, alle anderen Zeichen werden übernommen.
+////
+
+Beside the completion there is another script to display information about the git repository in the prompt. For this you have to load the file `contrib/completion/git-prompt.sh` (maybe it is also installed by your distribution, e.g. under `/usr/lib/git-core/git-sh-prompt`). Then, as in the following example, place a call to the `__git_ps1` function in the `PS1` variable. The function takes a so-called _format string expression_ as argument - i.e. the string `%s` is replaced by git infos, all other characters are taken over.
```

Comment:
The `__git_ps1` formatting causes the html text to look like this:

![screenshot](https://user-images.githubusercontent.com/18380976/101241879-57eff680-36fa-11eb-9bf2-791a727e00d4.png)